pkg/nfd-master/nfd-master.go

```diff
@@ -249,6 +239,18 @@ func (m *nfdMaster) Run() error {
 			if err := m.configure(m.configFilePath, m.args.Options); err != nil {
 				return err
 			}
+
+			// restart NFD API controller
+			if m.nfdController != nil {
+				klog.V(2).Infof("stopping the nfd controller")
```

Comment:
For symmetry I'd suggest to use the same log level as in the api controller startup message